lib/mix/lib/mix/tasks/app.start.ex

```diff
@@ -7,10 +7,10 @@ defmodule Mix.Tasks.App.Start do
   Starts all registered apps.
 
   First this task guarantees that all dependencies are in place
-  and that the current project has been compiled. Then the current
+  and that the current project has been compiled. Then, the current
   application is started as a temporary application, unless
   `:start_permanent` is set to `true` in your project configuration
-  or the `--permanent` option is given, then it's started as permanent,
+  or the `--permanent` option is given. Then, it's started as permanent,
   which guarantees the node will shut down if the application
   crashes permanently.
```

Comment:
```suggestion
  or the `--permanent` option is given. Setting it to permanent
  guarantees the node will shut down if the application terminates
  (typically because its root supervisor has terminated).
```